### PR TITLE
Fix stake page UI  desktop/mobile 

### PR DESCRIPTION
**Pull request by ONEONUORA**

### CODE REVIEW

File: frontend/public/background(jpeg).png
Comment (whateverfw) [inline frontend/public/background(jpeg).png:1]: seems like there are already existing background file.

File: frontend/src/pages/vault/stake/stake.css
Comment (whateverfw) [inline frontend/src/pages/vault/stake/stake.css:1]: remove commented code

Comment (whateverfw) [inline frontend/src/pages/vault/stake/stake.css:1]: this mobile part is broken  
image

File: frontend/src/pages/vault/vaultLayout.css
Comment (whateverfw) [inline frontend/src/pages/vault/vaultLayout.css:1]: remove commented code

File: frontend/src/pages/vault/stake/stake.css
Comment (whateverfw) [inline frontend/src/pages/vault/stake/stake.css:1]: remove commented code

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Comment by ONEONUORA** [comment]
Okay working on it

File: frontend/public/background(jpeg).png
Comment (whateverfw) [reply frontend/public/background(jpeg).png:1]: please use existing background file, delete this

**Review by whateverfw — CHANGES_REQUESTED** [review]
everything seems to be OK now, both deskop and mobile, please fix tablet:

image

**Comment by ONEONUORA** [comment]
Pls can you point to right image name

**Comment by whateverfw** [comment]
@ONEONUORA 
> Pls can you point to right image name

https://github.com/djeck1432/spotnet/blob/main/frontend/public/background.png
it was already used on this page

**Comment by whateverfw** [comment]
@ONEONUORA any updates?

**Comment by ONEONUORA** [comment]
@whateverfw Pls i need more time, I am refactoring the code to fix the tablet

**Review by whateverfw — APPROVED** [review]
lgtm

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**